src/drivers/libretro/libretro.c

```diff
@@ -957,6 +960,18 @@ void retro_get_system_info(struct retro_system_info *info)
    info->block_extract    = false;
 }
 
+float get_aspect_ratio(unsigned width, unsigned height)
+{
+  if (aspect_ratio_par == 1) {
+    aspect_ratio_type = NES_8_7_PAR;
```

Comment:
make **aspect_ratio_type** a local variable in this function since its not used anywhere else, or just dont use a variable and directly return the ration (just cast it if need be)

Reply:
tried to directly return the ratio and it works, but lgtm-com is complaining about the float variable not returning a value.